### PR TITLE
It was translated the file new_component_todo-item.md

### PR DESCRIPTION
**Pull request by dtgfranca**

### CODE REVIEW

File: new_component_todo-item.md [hunk 1]
```diff
@@ -1,16 +1,18 @@
-# New component todo-item
+# Novo componente todo-item
 
-We will create a new component which will be used for each todo item that is displayed on the list. It will be a simple component at first, but it will grow later on. What's important is that **it will get the todo item as an input from its parent component**. This way it can be a reusable component, and not rely directly on the application's data and state.
+Nós iremos criar um novo componente  que poderá ser usado para cada
```
Comment (cyz) [inline new_component_todo-item.md]: Hi @dtgfranca, this topic need adjust, maybe something like "Vamos criar um novo componente que será usado para cada item que é exibido na lista."

File: new_component_todo-item.md [hunk 2]
```diff
@@ -1,16 +1,18 @@
-# New component todo-item
+# Novo componente todo-item
 
-We will create a new component which will be used for each todo item that is displayed on the list. It will be a simple component at first, but it will grow later on. What's important is that **it will get the todo item as an input from its parent component**. This way it can be a reusable component, and not rely directly on the application's data and state.
+Nós iremos criar um novo componente  que poderá ser usado para cada 
+item  todo que é mostrado na lista. Será um componente simples no começo, mas irá crescer
+posteriormente. O que é importante é que **ele receberá  o item todo  como uma entrada para o componente pai**. Deste modo poderá ser um componente reusável, e não depende da aplicação diretamente.
```
Comment (cyz) [inline new_component_todo-item.md]: suggestion: "ele receberá  o todo item como uma entrada para o componente pai"

Comment (cyz) [reply new_component_todo-item.md]: "Deste modo poderá ser um componente reutilizável e que não dependerá da aplicação diretamente."

File: new_component_todo-item.md [hunk 3]
```diff
@@ -1,16 +1,18 @@
...
 
-Create a new component called `item`: 
+Criaremos um novo componente chamado `item`: 
 
 ```
 ng g c item -it
 ```
 
-You can see a new folder was created with the component files inside. 
+Vocẽ pode ver uma nova pasta criada com os arquivos  do componente.
```
Comment (cyz) [inline new_component_todo-item.md]: "Você"

File: new_component_todo-item.md [hunk 4]
```diff
@@ -20,25 +22,25 @@ Use the component in the template of `appComponent` - inside the `<li>` element:
 </ul>
 ```
 
-Check out the result in the browser. What do you see? Why?
+Confira o resultado no navegador. O que você  ver? Por que?
```
Comment (cyz) [inline new_component_todo-item.md]: "O que você vê? Por que?"

File: new_component_todo-item.md [hunk 5]
```diff
@@ -20,25 +22,25 @@ Use the component in the template of `appComponent` - inside the `<li>` element:
...
 
 ## @Input()
-We want to display the title of each item from within the `todo-item` component. We need to pass the title of the current item in the loop (or the whole item) to the `todo-item` component. 
+Nós queremos exibir o titulo de cada item dentro do componente `todo-item`. Precisamos passar o titulo atual do item  no loop(ou o item inteiro) para o o componente `todo-item`.
```
Comment (cyz) [inline new_component_todo-item.md]: "título"

File: new_component_todo-item.md [hunk 6]
```diff
@@ -20,25 +22,25 @@ Use the component in the template of `appComponent` - inside the `<li>` element:
...
 
-Check out the result in the browser. What do you see? Why?
+Confira o resultado no navegador. O que você  ver? Por que?
 
 ## @Input()
-We want to display the title of each item from within the `todo-item` component. We need to pass the title of the current item in the loop (or the whole item) to the `todo-item` component. 
+Nós queremos exibir o titulo de cada item dentro do componente `todo-item`. Precisamos passar o titulo atual do item  no loop(ou o item inteiro) para o o componente `todo-item`. 
 
-Again, Angular makes it really easy for us, by providing us the `Input` decorator.
+Novamente, Angular faz com que seja muito fácil para nós, fornecendo-nos o decoratore `Input`.
```
Comment (cyz) [inline new_component_todo-item.md]: "decorator" is a technical term and don't need translator

File: new_component_todo-item.md [hunk 7]
```diff
@@ -20,25 +22,25 @@ Use the component in the template of `appComponent` - inside the `<li>` element:
...
 
-Inside the newly generated `itemComponent` class (itemComponent) add the line:
+Dentro da classe recém criada `itemComponent` (itemComponent) adicione a linha:
 ```ts
 @Input() itemTitle: string;
 ```
-It tells the component to expect an input of type string and to assign it to the class member called `itemTitle`. Make sure that `Input` is added to the import statement in the first line in the file. Now we can use it inside the `itemComponent` template:
+Ele diz ao componente que espera uma entrada do tipo string e atribue ao membro da classe chamada `itemTitle`. Certifique-se  de que o `Input` iestá adicionado na declaração de importação na primeira linha do arquivo.Agora, podemos usá-lo  dentro do template `itemComponent`:
```
Comment (cyz) [inline new_component_todo-item.md]: Please, fix the phrase "Certifique-se  de que o `Input`" ...

File: new_component_todo-item.md [hunk 8]
```diff
@@ -20,25 +22,25 @@ Use the component in the template of `appComponent` - inside the `<li>` element:
...
 ```html
 {{ itemTitle }}
 ```
 
-You can add any other HTML elements you'd like here. 
+Vocẽ pode adicionar qualquer outro elemento HTML que você desejar aqui.
```
Comment (cyz) [inline new_component_todo-item.md]: pls, fix the first word

File: new_component_todo-item.md [hunk 9]
```diff
@@ -20,25 +22,25 @@ Use the component in the template of `appComponent` - inside the `<li>` element:
...
 
-Now we need to pass a string, which is the item's title, where we use the component. Go back to `appComponent` and  pass the item title to the `todo-item`:
+Agora precisamos passar uma String, qual é o titulo do item, onde nós usamos o componente. Volte ao `appComponent` e passe o titulo do item ao `todo-item`:
```
Comment (cyz) [inline new_component_todo-item.md]: pls, fix the word title in portuguese

File: new_component_todo-item.md [hunk 10]
```diff
@@ -47,23 +49,23 @@ Now we need to pass a string, which is the item's title, where we use the compon
 </ul>
 ```
 
-The `itemTitle` here in square brackets is the same as declared as the component's `@Input`.
+O `itemTitle` entre colchetes é o mesmo que foi declarado como componente no `@Input`.
 
-We use property binding on an element we created ourselves! And now we can actually see and understand that property binding binds to actual property of the component. 
+Nós usamos a propriedade Binding em um elemento que nós mesmo criamos! E agora podemos realmente ver  e entender que a vinculação da propriedade Binding se liga a propriedade atual do componente.
```
Comment (cyz) [inline new_component_todo-item.md]: pls, add plural state for "mesmo" and the second phrase to better understanding.

File: new_component_todo-item.md [hunk 11]
```diff
@@ -72,4 +74,4 @@ Now pass the whole item to the renamed property in `appComponent` (remove the `.
 </ul>
 ```
 
-We now have a list of components, so each component received its data from the loop in the parent component. Now we'll see how this list can be dynamic.
+Agora temos uma lista de componentes, então cada componente recebeu seus dados de um loop do componente pai. Agora, veremos  como esta lista pode ser dinâmica
```
Comment (cyz) [inline new_component_todo-item.md]: pls, add a final dot here

File: new_component_todo-item.md [hunk 12]
```diff
@@ -47,23 +49,23 @@ Now we need to pass a string, which is the item's title, where we use the compon
...
+O `itemTitle` entre colchetes é o mesmo que foi declarado como componente no `@Input`.
 
-We use property binding on an element we created ourselves! And now we can actually see and understand that property binding binds to actual property of the component. 
+Nós usamos a propriedade Binding em um elemento que nós mesmo criamos! E agora podemos realmente ver  e entender que a vinculação da propriedade Binding se liga a propriedade atual do componente.
 
-## Passing the whole item
-We will refactor our code a bit so we can easily implement more functionality in the `todo-item` component, for example editing and removing the item. Instead of passing just the title to the component, we will pass the whole item, and let the component extract the title where needed.
+## Passando o item inteiro
+Agora iremos refatorar um pouco nosso código para que possamos facilmente implementar mais funcionalidades no componente `todo-item`, por exemplo editando e removendo o item. Em vez de passar apenas o título ao componente, nós poderiamos passar todo o item, e deixamos o componente extrair o título quando necessário.
```
Comment (cyz) [inline new_component_todo-item.md]: Pls, check the comma and connection between sentences.

File: new_component_todo-item.md (hunk 1 above)
Comment (dtgfranca) [reply new_component_todo-item.md]: Thanks, topic adjusted.

File: new_component_todo-item.md (hunk 2 above)
Comment (dtgfranca) [reply new_component_todo-item.md]: Thanks, corrected.

File: new_component_todo-item.md (hunk 3 above)
Comment (dtgfranca) [reply new_component_todo-item.md]: Thanks, corrected.

File: new_component_todo-item.md (hunk 4 above)
Comment (dtgfranca) [reply new_component_todo-item.md]: Thanks, corrected #063c991.

File: new_component_todo-item.md (hunk 5 above)
Comment (dtgfranca) [reply new_component_todo-item.md]: Thanks, corrected 063c991.

File: new_component_todo-item.md (hunk 6 above)
Comment (dtgfranca) [reply new_component_todo-item.md]: Thanks, corrected #063c991.

File: new_component_todo-item.md (hunk 7 above)
Comment (dtgfranca) [reply new_component_todo-item.md]: Thanks, corrected #063c991.

File: new_component_todo-item.md (hunk 8 above)
Comment (dtgfranca) [reply new_component_todo-item.md]: Thanks, corrected #063c991.

File: new_component_todo-item.md (hunk 9 above)
Comment (dtgfranca) [reply new_component_todo-item.md]: Thanks, corrected #063c991.

File: new_component_todo-item.md (hunk 10 above)
Comment (dtgfranca) [reply new_component_todo-item.md]: Thanks, corrected #063c991.

File: new_component_todo-item.md (hunk 12 above)
Comment (dtgfranca) [reply new_component_todo-item.md]: Thanks, corrected #063c991.

File: new_component_todo-item.md (hunk 11 above)
Comment (dtgfranca) [reply new_component_todo-item.md]: Thanks, corrected #063c991.

**Comment by cyz** [comment]
@dtgfranca thank you so much for the fixes. PR approved! 👍

**Pull request merged**